src/main/resources/avro/bdg.avdl

```diff
@@ -286,6 +299,13 @@ record Fragment {
    */
   array<Sequence> sequences = [];
   array<AlignmentRecord> alignments = [];
+
+
+  /**
+   Platform-specific information about this Fragment.
+   */
+  union { null, Platform } platformType;
```

Comment:
Do Avro schemas need this extra bit of info or can you simply do a runtime type test on the `platformInfo` field?